literature.bib

```diff
@@ -2072,6 +2241,138 @@ @inproceedings{WKR:SPLC22
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2021 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@book{BHvMW21,
+	author = {Biere, Armin and Heule, Marijn and van Maaren, Hans and Walsh, Tony},
+	title = {{Handbook of Satisfiability}},
+	isbn = {978-1-64368-160-3},
+	publisher = IOS,
+	address = Amsterdam,
+	year = 2021,
+	ek-tags = {satisfiability},
+}
+
+@incollection{BJK21,
+	title = {{Preprocessing in SAT Solving}},
+	author = {Biere, Armin and J{\"a}rvisalo, Matti and Kiesl, Benjamin},
+	booktitle = {Handbook of Satisfiability},
+	publisher = IOS,
+	address = Amsterdam,
+	year = 2021,
+	ek-tags = {satisfiability, preprocessing},
+}
+
+@article{BRSS:COLA22,
+	title = {{Family-Based and Product-Based Development of Correct-by-Construction Software Product Lines}},
+	author = {Tabea Bordis and Tobias Runge and David Schultz and Ina Schaefer},
+	journal = COLA,
+	volume = 70,
+	pages = 101119,
+	year = 2022,
+	doi = {10.1016/j.cola.2022.101119},
+}
+
+@inproceedings{GEQR:SPLC21,
+	author = {Gu\'{e}gain, \'{E}douard and Quinton, Cl\'{e}ment and Rouvoy, Romain},
+	title = {{On Reducing the Energy Consumption of Software Product Lines}},
+	booktitle = SPLC,
+	publisher = ACM,
+	pages = {89–-99},
```

Comment:
Long stroke.
Have to check the script. Actually it is supposed to catch this issue :thinking:

Reply:
Fixed the script in d16bc47dc8c7d6b304e4a2f9573507eb0c36390c